### PR TITLE
Documentation alert

### PR DESCRIPTION
**Pull request by braswelljr**
## Description
Alert for New Documentation at [https://glab-cli.io](https://glab-cli.io)

## Related Issue

Resolves #[issue_number]

## Screenshots (if appropriate)
Screenshot from 2021-11-02 11-38-54
Screenshot from 2021-11-02 11-40-46

## Types of changes

- [X] Documentation
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (Related to CI or Packaging to platforms)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,9 +98,11 @@ Table of Contents
 
 ## Learn More
 Read the [documentation](https://glab.readthedocs.io/) for more information on this tool.
+### Documentation
+Get the new documentation on [`https://glab-cli.io`](https://glab-cli.io) and contribute to the new documentation on [https://github.com/braswelljr/glab-docs](https://github.com/braswelljr/glab-docs).
 
 ## Installation
-Download a binary suitable for your OS at the [releases page](https://github.com/profclems/glab/releases/latest).
+Download a binary suitable for your OS at the [releases page](https://github.com/profclems/glab/releases/latest)
```
Comment (profclems) [inline README.md:73]: We want to maintain the period at the end.

File: README.md [hunk 2]
```diff
@@ -98,9 +98,11 @@ Table of Contents
 
 ## Learn More
 Read the [documentation](https://glab.readthedocs.io/) for more information on this tool.
+### Documentation
+Get the new documentation on [`https://glab-cli.io`](https://glab-cli.io) and contribute to the new documentation on [https://github.com/braswelljr/glab-docs](https://github.com/braswelljr/glab-docs).
```
Comment (profclems) [inline README.md]: Remove the documentation header. 

I'm thinking maybe we could move the docs from the https://github.com/braswelljr/glab-docs repo to the /docs folder in this repo.

**Review by profclems — CHANGES_REQUESTED** [review]
Great work @braswelljr! 

Thanks so much for working on this. We may have to decide whether to move the docs from the https://github.com/braswelljr/glab-docs repo to the /docs folder in this repo or maintain it at the current repo.

**Comment by braswelljr** [comment]
I am glad my work has been approved.

I am ready to do what is best and help improve and make sure we deliver a
good and user friendly service to the open source community.

**Pull request closed without merging**